### PR TITLE
PPS pixel detector edges

### PR DESCRIPTION
**Pull request by fabferro**
#### PR description:
This PR reduces the sensitive dimension of the pixel detector at the edges by 50um.
It also introduce a bool method (isPixelHit) to check if x,y coordinates belong to the pixel detector area.

#### PR validation:
The new code has been privately tested over a set of ZeroBias events.
Only very minimal changes expected in the output of the reconstruction.

### CODE REVIEW

**Comment by fabferro** [comment]
@jan-kaspar @robutti @forthommel FYI

**Comment by civanch** [comment]
please test

File: Geometry/VeryForwardGeometry/interface/CTPPSPixelSimTopology.h [hunk 1]
```diff
@@ -60,7 +60,7 @@ class CTPPSPixelSimTopology : public CTPPSPixelTopology
 
       // rows (x segmentation)
       if ( arow == 0 ) {
-        lower_x = 0;
+        lower_x = 0.05;
```
Comment (cvuosalo) [inline Geometry/VeryForwardGeometry/interface/CTPPSPixelSimTopology.h]: This file is filled with magic numbers. They should be replaced with constants now or in a later pull request. See [7.6](https://github.com/cms-sw/cms-sw.github.io/blob/3e89200bde531a9aa1d01d4c92c1d65a5f6c87fd/cms_coding_rules.md#7----design-and-coding-guidelines-1) of our coding guidelines.

Comment (fabferro) [reply Geometry/VeryForwardGeometry/interface/CTPPSPixelSimTopology.h]: done

**Comment by cvuosalo** [comment]
+1

**Comment by cvuosalo** [comment]
please test

**Comment by fabiocos** [comment]
@fabferro do we have in the standard test workflows histograms that are supposed to be sensitive to this change?

**Comment by fabferro** [comment]
> 
> 
> @fabferro do we have in the standard test workflows histograms that are supposed to be sensitive to this change?

@fabiocos I don't think so. It's a fine tuning of the edge dimensions, so you need huge statistics to see a change in histograms.

**Comment by fabiocos** [comment]
+1

**Pull request merged**